pyaerocom/aeroval/experiment_processor.py

```diff
@@ -132,19 +132,19 @@ def run(self, model_name=None, obs_name=None, var_list=None, update_interface=Tr
         self.cfg._check_time_config()
 
         obs_list = self.cfg.obs_cfg.keylist(obs_name)
-        if not self.cfg.model_cfg:
+        model_list = self.cfg.model_cfg.keylist(model_name)
+        if not model_list:
             logger.info("No model found, will make dummy model data")
             self.cfg.webdisp_opts.hide_charts = ["scatterplot"]
             self.cfg.webdisp_opts.pages = ["evaluation", "infos"]
             model_id = make_dummy_model(obs_list, self.cfg)
+            model_list.append("dummy")
```

Comment:
What is the intent here? Does this relate to the preceeding line? Maybe a clarifying comment would be useful since this does not appear related to the preceeding lines?

Reply:
The line above adds the dummy to cfg, but not to the model_list, which is still empty. This list is used later. But I've added a comment